docs/src/userguide/glossary.rst

```diff
@@ -13,6 +13,16 @@ Glossary
       `allocated dynamically/heap allocated <https://en.wikipedia.org/wiki/C_dynamic_memory_allocation>`_.
       Its lifetime is until the user deletes it explicitly (with ``free`` in C or ``del`` in C++).
       This can happen in a different function than the allocation.
+      
+   Global Interpreter Lock or GIL
```

Comment:
For some reason it's linking to the CPython glossary on my PC rather than this definition. I'm not really sure why, but I'm hoping it's a local configuration problem.